client/k8sclient/k8sclient.go

```diff
@@ -85,23 +85,22 @@ func New(config Config) (kubernetes.Interface, error) {
 	if config.Address != "" {
 		_, err := url.Parse(config.Address)
 		if err != nil {
-			return nil, microerror.Maskf(invalidConfigError,
-				"config.Address=%s must be a valid URL: %s", config.Address, err)
+			return nil, microerror.Maskf(invalidConfigError, "config.Address=%s must be a valid URL: %s", config.Address, err)
```

Comment:
The other errors are not wrapped. I find the consistency better and also think this is more readable that way.